test/plotTestStatCLs.py

```diff
@@ -42,6 +43,13 @@
     type="str",
     help="poi values, comma separated (type all to make a plot for every value found in the file)",
 )
+parser.add_option(
+    "",
+    "--sub_label",
```

Comment:
I'm not sure if there is an existing convention for some of this combine scripts already. But (at least with recent versions) of option/argument parser, using `--some-command-line-option` name will automatically produce the variable using underscores rather than `-`, i.e. `some_command_line_option` in this case. Which  I think is just a little more user friendly, as it is a more often used convention.

(Same comment below)

Reply:
No strong feelings here so can go with hyphens instead